python/lsst/consdb/transformed_efd/dao/base.py

```diff
@@ -0,0 +1,380 @@
+"""Provides the `DBBase` class for managing database access."""
+
+import warnings
+from typing import Any, Dict, List
+
+import numpy
+import pandas
+from sqlalchemy import Engine, MetaData, Table, create_engine
+from sqlalchemy import exc as sa_exc
+from sqlalchemy import func
+from sqlalchemy.dialects import postgresql, sqlite
+from sqlalchemy.pool import NullPool
+
+
+class DBBase:
```

Comment:
Are there any other common utilities that could be used for something like this in ConsDB? (generic DAO implementation)

Reply:
I am not sure.